modules/cluster-logging-elasticsearch-exposing.adoc

```diff
@@ -15,23 +15,40 @@ log store CA certificate. Then, access a node that hosts the log store service w
 * The `Authorization: Bearer ${token}`
 * The Elasticsearch reencrypt route and an link:https://www.elastic.co/guide/en/elasticsearch/reference/current/api-conventions.html[Elasticsearch API request].
 
-Internally, you can access the log store service using the Elasticsearch cluster IP:
-
-You can get the log store service cluster IP using either of the following commands:
+Internally, you can access the log store service using the log store cluster IP,
+which you can get using either of the following commands:
 
+[source,terminal]
 ----
 $ oc get service elasticsearch -o jsonpath={.spec.clusterIP} -n openshift-logging
+----
 
+.Example output
+[source,terminal]
+----
 172.30.183.229
 ----
 ----
```

Comment:
There's no newline between these blocks. Is this the other _following_ command that can be used?